scripts/ci.sh

```diff
@@ -16,6 +16,7 @@ elif [ "$TEST" = "build_and_lint" ]; then
   npm run build
   npm run dtslint
   npm run depcheck
+  npx bundlesize
```

Comment:
Wouldn't it be correct to call the ``bundlesize`` checker with ``npm run bundlesize`` instead of the direct call with ``npx``?